### PR TITLE
Fix/cadless fixes svandenhoek

### PR DESCRIPTION
**Pull request by svandenhoek**
Adjustments for #49 based on partial review.

### New
- Test script to validate whether `convert_vep_vcf_to_tsv_capice.sh` returns valid output/expected exitcode.

### Changes
- `convert_vep_vcf_to_tsv_capice.sh` requires `.tsv.gz` as output file path (previously both `.tsv` and `.tsv.gz` were accepted, though the latter was bugged).
- Added additional improvements (such as stopping `grep` after finding first match) to `convert_vep_vcf_to_tsv_capice.sh`.
- Some methods made private as these shouldn't be called outside the class anyway.
- Replaced some (almost) duplicate code with shared method.
- Replaced several `.format()` strings with f-Strings.
- Adjusted some pydoc.

### Fixed
- Invalid input arguments for `convert_vep_vcf_to_tsv_capice.sh` should now be handled better prior to actually running BCFTools.
- Some minor bugs within the python code.

### CODE REVIEW

File: src/main/python/core/global_manager.py [hunk 1]
```diff
@@ -265,18 +265,18 @@ def grch_build(self):
...
         """
         return self._grch_build
 
     @grch_build.setter
     def grch_build(self, value):
         """
-        Singleton property grch_build, to set the globally available GRCh build
-        parsed from the config.
+        Singleton property grch_build,
+        to set the GRCh build present in the parsed input file.
```
Comment (SietsmaRJ) [inline src/main/python/core/global_manager.py:276]: False, this value is set after evaluating both the config and input file GRCh builds to be used in imputing and predicitng.

**Review by SietsmaRJ — CHANGES_REQUESTED** [review]

File: src/main/python/core/global_manager.py (hunk 1 above)
Comment (SietsmaRJ) [reply src/main/python/core/global_manager.py:276]: > `@grch_build.setter` -> sets final grch build (fix was invalid)

Correct

> `@config_grch_build.setter` -> "to set the GRCh build present in the config"

Correct

> `@vep_version.setter` -> "VEP version present in the parsed input file"

Incorrect, this is the same as grch_build.setter, but for the VEP version.

> `@config_vep_version.setter` -> "VEP version present in the config"

Correct.

Comment (svandenhoek) [reply src/main/python/core/global_manager.py:276]: Aaaah, then I fixed it the wrong way around. Thanks!

Comment (svandenhoek) [reply src/main/python/core/global_manager.py]: Commit is added which corrects it: https://github.com/molgenis/capice/pull/55/commits/c9427dfc84411493d3ce94beda07731a3c519e74

**Review by SietsmaRJ — APPROVED** [review]

**Pull request merged**